modules/cluster/main.tf

```diff
@@ -74,31 +74,31 @@ data "aws_iam_policy_document" "permissions" {
 
 module "asg" {
   source               = "telia-oss/asg/aws"
-  version              = "0.2.1"
+  version              = "v3.0.1"
   name_prefix          = "${var.name_prefix}-cluster"
-  user_data            = "${coalesce(var.user_data, data.template_file.main.rendered)}"
-  vpc_id               = "${var.vpc_id}"
-  subnet_ids           = "${var.subnet_ids}"
+  user_data            = coalesce(var.user_data, data.template_file.main.rendered)
+  vpc_id               = var.vpc_id
+  subnet_ids           = var.subnet_ids
   await_signal         = "true"
   pause_time           = "PT5M"
   health_check_type    = "EC2"
-  instance_policy      = "${data.aws_iam_policy_document.permissions.json}"
-  min_size             = "${var.min_size}"
-  max_size             = "${var.max_size}"
-  instance_type        = "${var.instance_type}"
-  instance_ami         = "${var.instance_ami}"
-  instance_key         = "${var.instance_key}"
-  instance_volume_size = "${var.instance_volume_size}"
-  ebs_block_devices    = "${var.ebs_block_devices}"
-  tags                 = "${var.tags}"
+  instance_policy      = data.aws_iam_policy_document.permissions.json
+  min_size             = var.min_size
+  max_size             = var.max_size
+  instance_type        = var.instance_type
+  instance_ami         = var.instance_ami
+  instance_key         = var.instance_key
+  instance_volume_size = var.instance_volume_size
+  ebs_block_devices    = var.ebs_block_devices
```

Comment:
New ASG module expects this to be `list(map(string))` [here](https://github.com/telia-oss/terraform-aws-asg/blob/master/variables.tf#L56)